compose/foundation/foundation/build.gradle

```diff
@@ -117,7 +117,7 @@ if (AndroidXComposePlugin.isMultiplatformEnabled(project)) {
             skikoMain {
                 dependsOn(commonMain)
                 dependencies {
-                    api(libs.skikoCommon)
+                    implementation(libs.skikoCommon)
```

Comment:
Native tests are failing

Reply:
Yep. We need to revisit our API because there are a few skiko types "leaks" into public